libyul/AssemblyStack.h

```diff
@@ -124,9 +124,9 @@ class AssemblyStack: public langutil::CharStreamProvider
 private:
 	bool analyzeParsed();
 	bool analyzeParsed(yul::Object& _object);
-
+public:
```

Comment:
check

Reply:
Hm... I'm actually using it for ``EVMCodeTransformTest``...